# FaultBit.msg - 故障位数据
# 故障位信息
string item             # 故障项目名称
int8 value              # 故障值 (0-正常, 1-故障)
string text             # 故障描述文本